### ISSUE TITLE
Installation failed with Strapi 4.10.1. React version 18

### ISSUE THREAD

**Issue by minht11** [ComfortablyCoding/strapi-plugin-publisher#71]
```
Found: react@18.2.0
npm ERR! node_modules/react
npm ERR!   peer react@"^17.0.0 || ^18.0.0" from @strapi/design-system@1.7.2
npm ERR!   node_modules/@strapi/design-system
npm ERR!     peer @strapi/design-system@"^1.6.6" from @strapi/helper-plugin@4.10.1
npm ERR!     node_modules/@strapi/helper-plugin
npm ERR!       peer @strapi/helper-plugin@"^4.0.7" from strapi-plugin-publisher@1.3.3
npm ERR!       node_modules/strapi-plugin-publisher
npm ERR!         strapi-plugin-publisher@"^1.3.3" from A@0.0.1
npm ERR!         site/api
npm ERR!   peer react@"^17.0.0 || ^18.0.0" from @strapi/icons@1.7.2
npm ERR!   node_modules/@strapi/icons
npm ERR!     peer @strapi/icons@"^1.6.6" from @strapi/helper-plugin@4.10.1
npm ERR!     node_modules/@strapi/helper-plugin
npm ERR!       peer @strapi/helper-plugin@"^4.0.7" from strapi-plugin-publisher@1.3.3
npm ERR!       node_modules/strapi-plugin-publisher
npm ERR!         strapi-plugin-publisher@"^1.3.3" from A@0.0.1
npm ERR!         site/api
npm ERR!     peer @strapi/icons@"^1.5.0" from @strapi/design-system@1.7.2
npm ERR!     node_modules/@strapi/design-system
npm ERR!       peer @strapi/design-system@"^1.6.6" from @strapi/helper-plugin@4.10.1
npm ERR!       node_modules/@strapi/helper-plugin
npm ERR!         peer @strapi/helper-plugin@"^4.0.7" from strapi-plugin-publisher@1.3.3
npm ERR!         node_modules/strapi-plugin-publisher
npm ERR!   3 more (react-dom, react-router-dom, styled-components)
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer react@"^17.0.2" from @strapi/helper-plugin@4.10.1
npm ERR! node_modules/@strapi/helper-plugin
npm ERR!   peer @strapi/helper-plugin@"^4.0.7" from strapi-plugin-publisher@1.3.3
npm ERR!   node_modules/strapi-plugin-publisher
npm ERR!     strapi-plugin-publisher@"^1.3.3" from A@0.0.1
npm ERR!     site/api
```

**Comment by Herraiz** [comment]
Same problem here :(

**Comment by ComfortablyCoding** [comment]
This should be fixed in v1.3.4 (latest release)
